[2024-04-17T16:34:40.623+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: files_ext.check_dir manual__2024-04-17T16:29:37.837702+00:00 [queued]>
[2024-04-17T16:34:40.631+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: files_ext.check_dir manual__2024-04-17T16:29:37.837702+00:00 [queued]>
[2024-04-17T16:34:40.632+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 4
[2024-04-17T16:34:40.644+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): check_dir> on 2024-04-17 16:29:37.837702+00:00
[2024-04-17T16:34:40.651+0000] {standard_task_runner.py:60} INFO - Started process 602 to run task
[2024-04-17T16:34:40.653+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'files_ext', 'check_dir', 'manual__2024-04-17T16:29:37.837702+00:00', '--job-id', '2061', '--raw', '--subdir', 'DAGS_FOLDER/files_extraction.py', '--cfg-path', '/tmp/tmp1wraijt4']
[2024-04-17T16:34:40.654+0000] {standard_task_runner.py:88} INFO - Job 2061: Subtask check_dir
[2024-04-17T16:34:40.691+0000] {task_command.py:423} INFO - Running <TaskInstance: files_ext.check_dir manual__2024-04-17T16:29:37.837702+00:00 [running]> on host fdd3cfdf3f53
[2024-04-17T16:34:40.757+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='files_ext' AIRFLOW_CTX_TASK_ID='check_dir' AIRFLOW_CTX_EXECUTION_DATE='2024-04-17T16:29:37.837702+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-17T16:29:37.837702+00:00'
[2024-04-17T16:34:40.759+0000] {logging_mixin.py:188} INFO - Directory already exists.
[2024-04-17T16:34:40.759+0000] {python.py:202} INFO - Done. Returned value was: ['dags', 'logs', 'data', 'plugins', 'config', 'webserver_config.py', '***.cfg']
[2024-04-17T16:34:40.778+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=files_ext, task_id=check_dir, execution_date=20240417T162937, start_date=20240417T163440, end_date=20240417T163440
[2024-04-17T16:34:40.827+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-17T16:34:40.841+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
